examples/projections/nongeo/polar-misc.py

```diff
@@ -0,0 +1,158 @@
+r"""
+Miscellaneous ways to use polar projections
+===========================================
+
+Polar projections allow plotting polar coordinate data (e.g. angle
+:math:`\theta` and radius *r*). 
+
+The full syntax for polar projections is:
+
+**P**\ *width*\ [**+a**]\ [**+f**\ [**e**\|\ **p**\|\ *radius*]]
+[**+r**\ *offset*][**+t**\ *origin*][**+z**\ [**p**\|\ *radius*]]
```

Comment:
Need a trailing `\` to avoid a whitespace between the two lines.
```suggestion
**P**\ *width*\ [**+a**]\ [**+f**\ [**e**\|\ **p**\|\ *radius*]]\
[**+r**\ *offset*][**+t**\ *origin*][**+z**\ [**p**\|\ *radius*]]
```